react/ecmascript/01 - spread/index.js

```diff
@@ -1,7 +1,54 @@
 import isArray from './utils';
 
-export function min(numbers) {
-  return Math.min(...numbers);
+export function min(...numbers) {
+  if (numbers.length !== 0) {
+    if (isArray(numbers[0])) {
+      return Math.min(...numbers[0]);
+    }
+    if (numbers.length === 1) {
```

Comment:
There's no need to check this because there's nothing special about the case where `min` gets passed only 1 parameter